Tetris_formal.py

```diff
@@ -156,159 +187,139 @@ def get_direction(self, color):
         ]
 
         return up, down, right, left
-        
-        
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (20,90,90)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/租房照片二.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
+        """
```

Comment:
Ditto. The description doesn't mean anything. Please briefly describe the function.